website/src/blogs/install-wordpress-using-dynamic-nfs-provisioner.md

```diff
@@ -0,0 +1,251 @@
+
+  #### Add helm repo for WordPress
+
+  - Use the below command to add helm repo for WordPress
+    ```sh
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    ```
+  - Once repo has been added successfully, update helm repo using the following command
+    ```sh
+    helm repo update
+    ```
+
+  #### Installing WordPress
+  Once the repo has been updated successfully, we can install WordPress referring to dynamic NFS provisioner storageclass. In this case storageclass will be openebs-kernel-nfs, if you have different storageclass then update the value of --set persistence.storageClass accordingly. WordPress also accepts various other parameters, for more information click [here](https://github.com/bitnami/charts/tree/master/bitnami/wordpress#parameters)
+
+  ```sh
+  helm install my-release -n wordpress --create-namespace \
+    --set wordpressUsername=admin \
+    --set wordpressPassword=password \
+    --set mariadb.auth.rootPassword=secretpassword \
+    --set mariadb.primary.persistence.enabled=true \
+    --set mariadb.primary.persistence.storageClass=openebs-hostpath \
+    --set mariadb.primary.persistence.accessModes={ReadWriteOnce} \
+    --set persistence.storageClass=openebs-kernel-nfs \
+    --set persistence.accessModes={ReadWriteMany} \
+    --set volumePermissions.enabled=true \
+    --set autoscaling.enabled=true \
+    --set autoscaling.minReplicas=2 \
+    --set autoscaling.maxReplicas=6 \
+    --set autoscaling.targetCPU=80 \
+    bitnami/wordpress
+  ```
+
+  Above helm installation should create following resources in the cluster:
+
+  - Two WordPress application pod instances with RWX persistent volume.
+  - One Mariadb application pod instance with RWO persistent volume.
+
+  Results of helm wordpress installation:
+  ```sh
+  kubectl get po -n wordpress
+  NAME                                    READY   STATUS    RESTARTS   AGE
+  my-release-mariadb-0                    1/1     Running   0          2m34s
+  my-release-wordpress-85dc688bb9-24jx2   1/1     Running   0          2m34s
+  my-release-wordpress-85dc688bb9-wkfpd   1/1     Running   1          2m18s
+  ```
+
+  Once the installation process is succeeded, follow the steps mentioned in output of _helm install_ command to access WordPress from your browser:
+
+  [WordPress login page](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-login-page.png)
+
+  Now it's time to hack WordPress:
+
+  [WordPress landing page](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-landing-page.png)
+
+**Deployment View**:
+
+  [WordPress Deployment view](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-deployment-view.jpg)
```

Comment:
```suggestion
  ![WordPress login page](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-login-page.png)

  Now it's time to hack WordPress:

  ![WordPress landing page](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-landing-page.png)

**Deployment View**:

  ![WordPress Deployment view](/images/blog/install-wordpress-using-dynamic-nfs-provisioner-deployment-view.jpg)
```

Without exclamation mark it will appear as a link.
<img width="1440" alt="Screenshot 2021-11-10 at 11 08 11 AM" src="https://user-images.githubusercontent.com/34312966/141056187-1160dd94-3d0d-4c8e-afdc-5119f21cc150.png">

Reply:
Sure!! I got confused with this [comment](https://github.com/openebs/website/pull/288#discussion_r745486247)